### DISCUSSION TITLE
change the transmission url that the web proxy gives you

### DISCUSSION THREAD

**Discussion by RME-Systems** [AzuraCast/AzuraCast#5029] [Q&A] [answered]
Hello, I have an azuracast installation with Docker.
I enabled proxy in azuracast settings. The transmission link is:
https://radio.rmesystems.com/radio/8000/stream

I would like to know if it is possible to change that url for example to:
https://radio.rmesystems.com/stream
And if possible, how can I do it?
Thanks a lot.

**Comment by BusterNeece** [comment, marked as answer]
@RME-Systems It's not currently possible to provide a custom URL for the stream via the main web port, but we're looking into ways to make it possible in the future.
